dali/operators/decoder/nvjpeg/nvjpeg_decoder_decoupled_api.h

```diff
@@ -74,7 +78,7 @@ class nvJPEGDecoder : public Operator<MixedBackend>, CachedDecoderImpl {
     nvjpeg2k_thread_(1,
                      spec.GetArgument<int>("device_id"),
                      spec.GetArgument<bool>("affine")) {
-#if NVJPEG_VER_MAJOR > 11 || (NVJPEG_VER_MAJOR == 11 && NVJPEG_VER_MINOR >= 1)
+#if IF_HW_DECODER_COMPATIBLE
```

Comment:
This double `if` looks weird.
```suggestion
#if IS_HW_DECODER_COMPATIBLE
```

Reply:
Done